src/main/resources/reference.conf

```diff
@@ -201,10 +237,10 @@ cassandra-query-journal {
   
   # How many events to fetch in one query (replay) and keep buffered until they
   # are delivered downstreams.
-  max-buffer-size = 100
+  max-buffer-size = 10000
   
   # The fetch size of the Cassandra select statement
-  fetch-size = 50
+  fetch-size = 0
```

Comment:
This fetch-size is used in individual Statements in read journal and 0 tells the driver not to override the globally set value (http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/Statement.html). We set the global value (http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/QueryOptions.html) in CassandraPluginConfig. The default value in Java driver is 5000, but we override that in reference.conf, max-result-size = 50001. So simply 0 on Statement tells the driver to use 50001.

That may be too much for read journal, but may be a reasonable value for replay (if I am not mistaken it is what was used before this PR). Maybe we want to override the value for replay only for performance purposes?